packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -25,10 +28,10 @@ export interface ActionButtonsProps {
   onUpvoteClick?: (post: Post) => unknown;
   onCommentClick?: (post: Post) => unknown;
   onShare?: (post: Post) => unknown;
+  onBookmark?: (post: Post) => unknown;
```

Comment:
Perhaps we should keep the naming similar to `onBookmarkClick`

Reply:
Cool, I based this off the `onShare` which didn't have the `Click` part, but we can do that here.